docs/docs/integrations/embedding-stores/pgvector.md

```diff
@@ -17,12 +17,63 @@ https://github.com/pgvector/pgvector
 </dependency>
 ```
 
+## Gradle Dependency
+
+```implementation 'dev.langchain4j:langchain4j-pgvector:0.36.2'```
 
 ## APIs
 
 - `PgVectorEmbeddingStore`
 
+## Parameter Summary
+
+| Plain Java Property      | Spring Application Property | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Default Value       | Required/Optional                                                                                                                                                                                                                                                                                |

```

Comment:
Let's remove spring column for the time being, it only pollutes the table right now

Reply:
done